### ISSUE TITLE
BeatNet and 1D models latencies  

### ISSUE THREAD

**Issue by mjhydri** [WildHoneyPie/BEAST#1]
Hi there,

Great work! I just came across your paper and enjoyed reading it a lot. However, I noticed an error that I thought it may be worth mentioning. According to the paper, the latencies in the evaluation table are defined as the hop sizes of the other models. Therefore, the latencies for the BeatNet and 1D state space should be 20 ms as suggested in the papers and the implementations. However, they are reported as 46 ms in the evaluation.

For instance, the BeatNet model's hop size is defined on line [#62](https://github.com/mjhydri/BeatNet/blob/main/src/BeatNet/BeatNet.py) of the BeatNet inference main script. 

Just wanted to bring this error to your attention. Thank you for your valuable work.

Cheers,
Moji

**Comment by WildHoneyPie** [comment]
Hi Moji,

Thank you for your valuable previous works that have been a huge help in my
research.

I am sorry for the mistake that I made in the paper. I find the reported
hop size in section 2.1 of BeatNet <https://arxiv.org/pdf/2108.03576.pdf>
is 46 ms. And I didn't notice it was updated in the implementation and 1D
state space. Thank you for pointing out the error. I will modify the
reported latency on the paper submitted to the arXiv. Regarding its
acceptance by ICASSP, I'm exploring options to amend the error despite its
acceptance status.

Thank you once again for your support!

Best,
CC

Mojtaba (Moji) Heydari ***@***.***> 於 2024年1月2日 週二 下午4:05寫道：

**Comment by mjhydri** [comment]
Thank you very much for your prompt response and I am glad that my works were helpful in your research.

Regarding the reported BeatNet hop size you are absolutely right. As we started with the Bock model as the baseline, the hop was initialized with 46 ms. However, in the implementation phase, we lowered it to 20 ms for real-time practicality without modifying the reported number in the paper draft. However, In the 1D state space model, I made sure to report the correct hop size which is 20 ms.  

Thanks for addressing this issue and I appreciate your prompt attention to the matter. Keep up the great contributions!

Moji
